API-Samples/Tutorial/markdown/10-init_render_pass.md

```diff
@@ -105,10 +111,28 @@ optimal layout so that the GPU can read it more efficiently.
 Conversions from one layout to another are called layout transitions
 in Vulkan.
 You have control over these transitions and can invoke them
-with an image memory barrier command or as a subpass dependency in a render
-pass.
-In this case, the sample code uses image memory barrier commands, which
-are detailed later in this section.
+in one of three ways:
+
+1. Memory Barrier Command (via `vkCmdPipelineBarrier`)
+1. Render Pass specification
```

Comment:
I made that change to the list of ways.  And I also spelled it out more in the three paragraphs that follow the RenderPassLayouts diagram.